app/models/binary/InstallScript.scala

```diff
@@ -0,0 +1,218 @@
+       |
+       |	config_values[key_store_path]=wKKeyStore.jks
+       |	config_values[key_store_password]=$$password
+       |fi
+       |
+       |echo "In case the HTTPS setup fails, you can always change the play.server.https.keyStore values in the my-datastore.conf file. If you need more information regarding the HTTPS setup, please consult the documentation."
+       |
+       |echo "Generating config file"
+       |touch my-datastore.conf
+       |rm my-datastore.conf
+       |
+       |echo "include \\"/standalone-datastore.conf\\"" >> my-datastore.conf
+       |
+       |for i in "$${!config_values[@]}"
+       |do
+       |	echo "$${config_keys[$$i]} = \\"$${config_values[$$i]}\\"" >> my-datastore.conf
+       |done
+       |
+       |echo "Generated config file"
+       |
+       |echo "Additionally, webKnossos can be installed as a systemd service. This allows startup on boot and restarts on failure. To configure the service, you'll need root privileges."
+       |echo "Do you want to install webKnossos as a systemd service? Type [y] or [n]"
+       |
+       |read -n 1 install_service
+       |echo
+       |
+       |if [[ $$install_service == "y" ]]; then
+       |	service_file_definition="
+       |[Unit]
+       |Description=webKnossos datastore module
+       |
+       |Wants=network.target
+       |
+       |[Service]
+       |Type=simple
+       |WorkingDirectory=$$install_path
+       |ExecStart=$$install_path/run.sh
+       |Restart=on-failure
+       |RestartSec=10
+       |KillMode=mixed
+       |
+       |[Install]
+       |WantedBy=multi-user.target
+       |"
+       |
+       |	echo $$service_file_definition > webknossos-datastore.service
+       |	chmod 640 webknossos-datastore.service
+       |	sudo chown root webknossos-datastore.service
+       |	sudo chgrp root webknossos-datastore.service
+       |	sudo mv webknossos-datastore.service /etc/systemd/system/
+       |	sudo systemctl daemon-reload
+       |
+       |	echo "Do you wish to enable startup on boot? Type [y] or [n]"
+       |	read -n 1 startup_on_boot
+       |	echo
+       |
+       |	if [[ $$startup_on_boot == "y" ]]; then
+       |		sudo systemctl enable webknossos-datastore.service
+       |	fi
+       |
+       |	echo "Do you wish to start the service now? You can always start the service later by calling \"sudo systemctl start webknossos-datastore.service\". Type [y] or [n]"
+       |	read -n 1 start_now
+       |	echo
+       |
+       |	if [[ $$start_now == "y" ]]; then
+       |		sudo systemctl start webknossos-datastore.service
+       |	fi
+       |
+       |	echo "Service installation completed. To inspect the logs of the running webKnossos datastore module, you can use \"journalctl -u webknossos-datastore\" ."
+       |
+       |fi
+       |
+       |echo "Install completed! Have fun using your own webKnossos datastore."
+       |""".stripMargin
+}
```

Comment:
I am not very familiar with bash, some things I noticed here are
 - We rely on things like sudo and apt being present, and the user having bash. Is this risky? At least it should be communicated in the wk UI where the user downloads the script
 - Should we add default options for the yes-no-questions so that enter gets you to a standard case?
 - What can a user do after aborting the script mid-way? Does removing some generated directories restore a clean state so that they can start over? Should we explain this somewhere?
 - I’d ask @jstriebel to have a look at the bash script please, as I’m not very familiar with the language
 - Maybe @philippotto or @normanrz could comment on the phrasing / wording / what would you expect from this script.

Reply:
I just had a rough first look, this looks really good already. For better portability rather use `#!/usr/bin/env bash` than `/bin/bash` as the shebang. Checking for sudo and apt being present upfront (e.g. using `which`) and outputting a helpful error message would be nice.
Except for that, please check it also with https://www.shellcheck.net, this is quite helpful.

Reply:
PS: Please link to the respective documentation pages when referring to them

Reply:
> Maybe @philippotto or @normanrz could comment on the phrasing / wording / what would you expect from this script.

No strong opinion. My 2 cents:

`Installation completed! Your own webKnossos datastore is now ready to use. In case you experience any problems, please contact hello@scm.io.`

Reply:
Thanks for your input :slightly_smiling_face: 
@jstriebel I implemented all suggestion from shellcheck. Could you have another look? Regarding the documentation, I did not find any documentation for the config files. How would you communicate to the users that they need to look into the standalone conf file for all options @fm3?

> We rely on things like sudo and apt being present, and the user having bash. Is this risky? At least it should be communicated in the wk UI where the user downloads the script.

There are only a few places, where we need either of these present and it is communicated before, so I do not think that there is need for change.

> Should we add default options for the yes-no-questions so that enter gets you to a standard case?

Interesting question :thinking: I am a little concerned that the user would rush through the installation without reading some important questions. In the standard case (auto SSL and no service), the user has to input 7 things with 4 of them single key strokes. I feel this should not be a huge burden. However, I can try to add a default path, if you want.

> What can a user do after aborting the script mid-way? Does removing some generated directories restore a clean state so that they can start over? Should we explain this somewhere?

This script should be idempotent, so there is no work required after aborting. I added a comment in the first line about this.

Feel free to have another look :slightly_smiling_face:

Reply:
> How would you communicate to the users that they need to look into the standalone conf file for all options @fm3?

I think it’s fine not to explain the config format specifically. This setup is (for now) for users who don’t want particularly specific config options but a quick test.

> This script should be idempotent, so there is no work required after aborting.

Awesome!

> I am a little concerned that the user would rush through the installation without reading some important questions

Fair enough, feel free to keep it then.

> Feel free to have another look

Thanks, I’ll do that later today!

Reply:
I just had another look, LGTM :+1: